.github/workflows/test_periodic.yml

```diff
@@ -131,12 +110,7 @@ jobs:
           msbuild mysql-server-core-* php-* php7* \
           powershell temurin-* zulu-*
 
-    - name: Install KIND
-      run: |
```

Comment:
not sure I understand this change.. how do the tests run in periodic lanes without KIND install? I wonder if you can get a signal based on this PR for the periodic jobs to see they don't break since we don't run them as presubmits.

Reply:
This step does nothing useful.  It downloads a kind binary from aojea's repo.  This binary is not used since the later `kind setup` step will install a proper official version of kind.

Reply:
yeah you are right! resolved. thanks